starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -60,8 +60,12 @@ public void create() {
     inventory.setName(NAME);
     inventory.setProductType(PRODUCT_TYPE);
     inventoryDAO.create(inventory);
+    Inventory inventoryTwo = new Inventory();
```

Comment:
You're going to need to change this back to the way it was before. You accidentally overwrote it with this branch.